src/components/WwwFrame/EndOfYearAppealBanner/AppealImage.vue

```diff
@@ -0,0 +1,19 @@
+<template functional>
+	<img class="appeal-img">
+</template>
+
+<style lang='scss'>
+@import 'settings';
+
+.appeal-img {
+	width: 7.5rem;
+	height: 7.5rem;
+	border-radius: 50%;
+	background-position-y: center;
+	background-position-x: center;
+	background-repeat: no-repeat;
+	background-size: cover;
+	vertical-align: middle;
+	background-image: url('../../../assets/images/premal.jpg');
```

Comment:
this should use the short syntax... `url('~@/assets/images/customDropdown999.png');`